MAUI/DataForm/Editors.md

```diff
@@ -1318,6 +1332,137 @@ public class DataFormItemsSourceProvider : IDataFormSourceProvider
 
 N> [View editors sample in GitHub](https://github.com/SyncfusionExamples/maui-dataform/tree/master/DataFormEditors)
 
+## Segment editor
+
+In the `Segment` editor, the [SfSegmentedControl](https://www.syncfusion.com/maui-controls/maui-segmented-control) will be loaded.
+
+#### Changing the ItemsSource of Segment
+
+The [ItemsSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormListItem.html#Syncfusion_Maui_DataForm_DataFormListItem_ItemsSource) for the radio group editor is generated for the `enum` data type property. In order to add the `Segment` editor in the DataForm, you need to register the editor as `DataFormEditorType.Segment` for the required property by using the [RegisterEditor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_RegisterEditor_System_String_Syncfusion_Maui_DataForm_DataFormEditorType_) method.
```

Comment:
prepare the content properly, why radio group editor come in the segment control